content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/ios.md

```diff
@@ -533,3 +533,4 @@ The following attributes in `Trace.Configuration` can be used when creating the
 [9]: https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table
 [10]: https://docs.datadoghq.com/tracing/
 [11]: https://support.apple.com/guide/security/security-of-runtime-process-sec15bfe098e/web
+[12]: /tracing/trace_collection/custom_instrumentation/ios/otel
```

Comment:
why you're not adding the link to `opentelemetry` website here as for opentracing ?